### PR TITLE
Implement Brewing

### PR DESCRIPTION
**Pull request by CortexPE**
Title says it all ¯\\\_(ツ)_/¯

## TODO:
 - [X] Brewing Stand drops items
 - [X] Fix BrewingStands not sending items to the client
 - [X] Fuel System
 - [X] Some enhancements to make it more Vanilla-Like

### CODE REVIEW

**Comment by CortexPE** [comment]
shut the fuck up codacy -_-
I'm gonna disable you rn. too strict >.<

**Comment by CortexPE** [comment]
eh rip disabled bu the reviews can't removed... ripp

File: src/CortexPE/Main.php [hunk 1]
```diff
@@ -341,8 +341,8 @@ private function loadEverythingElse(){
 		// LevelManager::init(); EXECUTED VIA EventListener
 		Tile::init();
 		FishingLootTable::init();
-		PacketManager::init();
-		PacketManager::init();
+		//PacketManager::init();
+		(new BrewingManager())->init();
```
Comment (NycuRO) [inline src/CortexPE/Main.php]: THAT ACTUALLY WORKS LOL?

**Comment by TheAz928** [comment]
You forgot a thing. Brewing items can be taken by hoppers, so if you don't edit mechanics of hopper, it'll take items while brewing

**Comment by CortexPE** [comment]
Didn't forget about that 😂 
Coming on next commit... I consider it as part of the aesthetics

**Pull request merged**